metaserver/src/test/scala/tests/compiler/CompletionsTest.scala

```diff
@@ -108,15 +112,75 @@ object CompletionsTest extends CompilerSuite {
     """
       |case class User(name: String, age: Int)
       |object a {
-      | User("", 1).<<>>
+      |  User("", 1).<<>>
       |}
     """.stripMargin, { completions =>
       val completionLength = completions.items.length
       assert(completionLength > 2)
-      val completionLabels = completions.items.map(_.label)
-      assert(completionLabels.contains("name"))
-      assert(completionLabels.contains("age"))
+      val name = completions.items.find(_.label == "name")
+      val age = completions.items.find(_.label == "age")
+      assert(name.isDefined)
+      assert(age.isDefined)
+      assert(name.get.kind == Some(CompletionItemKind.Field))
+      assert(age.get.kind == Some(CompletionItemKind.Field))
     }
   )
 
+  check(
+    "trait",
+    """
+      |trait TestTrait
+      |object a {
+      |  val x: TestTr<<>>
+      |}
+    """.stripMargin,
+    s"""
+       |{
+       |  "isIncomplete" : false,
+       |  "items" : [ {
+       |    "label" : "TestTrait",
+       |    "kind" : ${CompletionItemKind.Interface},
+       |    "detail" : " extends "
```

Comment:
Is this detail expected? Seems a bit weird.

Reply:
it's weird but consistent to what I see in the editor. I suspect it's `Template`'s fault. I want to improve details in a future PR (Java LSP seems to be doing a nice job that we can mimic)